src/components/node-list/styles/_filter.scss

```diff
@@ -3,6 +3,31 @@
 @import '../../../styles/variables';
 @import './variables';
 
+//--- Text label ---//
+
+.pipeline-nodelist__row--unchecked {
+  // Fade row text when unchecked
+  .pipeline-nodelist__row__label--kind-filter {
+    opacity: 0.55;
```

Comment:
Perhaps it is worth to also set up a variable system for the text label opacity like what we have below?

Reply:
Can do if you think it necessary, but I didn’t set up a similar comment table or variable schema for the visibility (“toggle”) icons, because the CSS there is pretty simple and easy to understand, while on the filter icons it’s complex and hard to follow, so I found the table made it easier to refer to when refactoring the filter icons.

Reply:
also: ah I understand now! These rules explicitly _do not_ apply to the visibility icons, that’s why they’re on classes for the tag (“filter”) icons only.